content/articles/domains-au.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: .AU Domains
+excerpt: This article explains the requirements and special procedures for .AU domain names.
+categories:
+- Domains
+---
+
+# .AU Domain Names
+
+* TOC
+{:toc}
+
+---
+
+This article explains the requirements and special procedures for `.AU` domain names. DNSimple supports registration, renewal, and transfer of `.COM.AU`, `.ORG.AU`, and `.NET.AU` secondary top level domains.
+
+## General requirements
+
+All `.AU` domain names:
+
+-  require an Eligibility ID type and valid Eligibility ID number. Acceptable Eligibility ID types are:
+  - Australian Business Number
+  - Australian Company Number
+  - Trademark Number
+- require an Australian address.
+- must be an exact match, abbreviation, or acronym of the Registrant's name or otherwise closely and substantially connected to the Registrant.
+
+<warning>
+The organization name has to match the Australian Business Number (ABN) or Australian Company Number (ACN) information submitted.
+</warning>
+
+## Secondary top level domain requirements
+
+### `.COM.AU` and `.NET.AU`
+
+To register a `.COM.AU` or `.NET.AU` domain name, registrants must be one of the following:
+
+- An Australian registered company.
+- Trading under a registered business name in any Australian State or Territory.
+- An Australian partnership or sole trader.
+- A foreign company licensed to trade in Australia.
+- The owner of an Australian Registered Trademark.
+- An applicant for an Australian Registered Trademark.
+- An association incorporated in any Australian State or Territory.
+- An Australian commercial statutory body.
+
+### `.ORG.AU`
+
+To register a `.ORG.AU` domain name, registrants must belong to one of the following:
+
+- An association incorporated in any Australian State or Territory.
+- A political party registered with the Australian Electoral Commission.
+- A trade union or other organization registered under the Workplace Relations Act 1996.
+- A sporting or special interest club operating in Australia.
+- A charity operating in Australia, as defined in the Registrant's constitution or other documents of incorporation.
+- A non-profit organization operating in Australia, as defined in the Registrant's constitution or other documents of incorporation.
+
+<warning>
+A valid Australian Business Number (ABN) is required to register a `.ORG.AU` domain name. Australian Company Numbers and Trademark Numbers are not acceptable.
+</warning>
+
+## Renewals
+
+`.AU` domains must be renewed no later than 10 days before the expiration date. Automatic and manual renewals are permitted as long as the domain is less than 90 days away from expiry. If the domain name is not renewed by the deadline, it will enter [redemption](/articles/what-happens-when-domain-expires/#redemption-period). The domain name is removed from the DNS and cannot be updated. This period lasts for 31 days; during this period, the domain name can be recovered for an additional fee.
```

Comment:
> Automatic and manual renewals are permitted as long as the domain is less than 90 days away from expiry.

Unfortunately, we are limited by our current partner and we can't support manual renewal.

Reply:
@weppos wait, these tlds are auto renew only?

Reply:
At the time being yes, .AU is auto-renewal only. You cannot manually renew it.

Reply:
Ahh, I see. Should the suffixes be updated? The name tool indicates otherwise.

Reply:
After review, my comment is no longer valid.